src/lib/yang/parser.lua

```diff
@@ -189,7 +204,18 @@ end
 function Parser:parse_string()
    if self:check("'") then return self:parse_qstring("'")
    elseif self:check('"') then return self:parse_qstring('"')
-   else return self:take_while("[^%s;{}\"'/]") end
+   else
+      local ret = ""
+      repeat
+         ret = ret .. self:take_while("[^%s;{}\"'/]")
+         if self:is_eof() then break end
+         if self:peek_n(2) == "/*" then break end
+         if self:check("/") then
+            ret = ret .. "/"
+         end
+      until not self:check_pattern("[^%s;{}\"'/]")
+      return ret
```

Comment:
Ah yes, I did wonder about a lua equivalent of python's `"".join(list)`. Thanks for catching it.